internal/logger/logger.go

```diff
@@ -0,0 +1,41 @@
+package logger
+
+import (
+	"log"
+
+	"go.uber.org/zap"
+)
+
+var (
+	Logger *zap.Logger
+	Cfg    = config{}
+)
+
+type config struct {
+	Environment        string `env:"ENVIRONMENT" envDefault:"dev"`
+	DiscordServerID    string `env:"DISCORD_SERVER_ID"`
+	DiscordBotToken    string `env:"DISCORD_BOT_TOKEN"`
+	DiscordBotAsPublic bool   `env:"DISCORD_BOT_AS_PUBLIC" envDefault:"true"`
+}
```

Comment:
Maybe we can move this to `internal/config/config.go` instead?

Or if we want to do something flat we can do something like:

```
internal/
    config.go
    logger.go
```

And we would have references like:

```go
internal.Cfg
internal.Logger
```

This would also take care of the other comment

What do big projects do for this?

Reply:
I was thinking of moving all the types into a Type package. but I like the idea of moving the config into it's own pacakge.

Reply:
done.

Reply:
Beautiful, let's go